src/bosh-director-core/spec/bosh/director/core/templates/job_template_loader_spec.rb

```diff
@@ -79,7 +79,7 @@ module Bosh::Director::Core::Templates
 
       it 'returns the jobs template erb objects' do
         tarball_path = create_job_tarball(
-          'release-job-name',
+          'plop-job-name',
```

Comment:
Is there a reason for the change from `release` => `plop`? If this is a meaningless identifier maybe use something like `FAKE_XYZ...`

Reply:
Yep, “plop” just conventionally mean “_whatever name_” and is meant to be more fun to use than “fake”. “plip” is quite the same, maning “_whatever name, but different from ‘plop’_”.

All right for favoring “fake” over “plip/plop” 👍

Reply:
I kinda figured this was the case. I think over time we've found that  "fun" names tend to be less helpful that boring descriptive names. Sadly.